content/blog/2017-12-16-sourcebots-2018-rules.md

```diff
@@ -0,0 +1,14 @@
+---
+title:  SourceBots 2018 Rules
+date: 2017-12-16
+---
+
+After a very successful [kickstart][kickstart], the [rules for SourceBots
+2018][rules] have now been released.
+
+For those who were unable to attend kickstart, the [SB2018 Kickstart
+Presentation][kickstart-presentation] is also available.
+
+[kickstart]: /events/2018/kickstart
+[kickstart-presentation]: http://sourcebots.co.uk/wp-content/uploads/sites/244/2017/12/SB2018-Kickstart-Presentation.pptx
```

Comment:
I think this URL will cause issues. We have a public directory in google drive which we can use to serve things like this (Because putting things like that in Git doesnt make me feel right)

Reply:
I've no objection to putting stuff like this in git, these urls are pretty much placeholder -- I'm not sure how best to include them with hugo. If our standard practise is to put them in google drive, I'm happy to do that.

Reply:
As this presentation was originally done as a google sheets presentation anyway, I think linking to in google drive is the nicest solution.

Reply:
The other alternative I'd wondered about was a PDF (which IMO is a nicer thing to share than a PowerPoint file, though it loses the animations and slide notes).

Is there a way to share the presentation via a link that is read-only? I don't think we want people to be able to arbitrarily edit the presentation.

Reply:
I certainly agree we don't want them editing it. We can create read-only links to individual files yes very easily.

Reply:
7c0d997 switches to a google drive link.